<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Забота о глазах: Параметры</string>
  </property>
  <property name="toolTipDuration">
   <number>-3</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Расположение окна уведомлений</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Смещение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_of_panel">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Сторона</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_pos_of_panel"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sound_groupBox">
     <property name="title">
      <string>Звук уведомления</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="radioB_default_songs">
            <property name="text">
             <string>Из стандартных мелодий</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_song"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioB_user_songs">
            <property name="text">
             <string>Из Ваших</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="choice_user_song_button">
            <property name="text">
             <string>Обзор</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Громкость</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="valume">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Уведомлять каждые </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Длительность перерыва</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QSpinBox" name="interval_m_spin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rest_h_spin">
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>мин.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>ч.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QSpinBox" name="interval_s_spin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rest_m_spin">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>сек.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>мин.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_notif_face_out_of_camera">
     <property name="text">
      <string>Уведомлять о том, что лицо не может быть обнаружено</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
